--- main.c	2013-04-10 22:42:32.000000000 +0400
+++ ../../../../pixelkit_current/src/main.c	2013-02-17 23:06:18.000000000 +0400
@@ -45,7 +45,7 @@
 
 #define MAX_REPORTS	8
 
-PROGMEM int usbDescriptorStringSerialNumber[]  = {
+const PROGMEM int usbDescriptorStringSerialNumber[]  = {
  	USB_STRING_DESCRIPTOR_HEADER(4),
 	'1','0','0','0'
 };
@@ -261,7 +261,7 @@
 			break;
 		case 3:
 			curGamepad = snesGetGamepad();
-//#define NOMOUSE		
+#define NOMOUSE		
 #ifndef NOMOUSE
 			// auto-detect snes mouse
 			
